src/extrinsic/mod.rs

```diff
@@ -22,8 +22,7 @@ pub mod balances;
 pub mod contract;
 pub mod xt_primitives;
 
-pub extern crate codec;
-pub extern crate log;
```

Comment:
Has this been done for a reason?

I made this for issue #16 . This re-exports the dependencies locally. Making them available in the macro via `crate::...`.

Then third parties using `substrate-api-client` do not need to import the `codec` and `log` dependencies

Reply:
the reason for removing it is that it breaks `no_std`

Reply:
try reexporting conditionally, only if in `std` environment